salesagility/SuiteCRM
# Fixed - user names in the calendar titles

## Description




1. Refactored the way the calendars are constructed.
2. Fixed a minor CSS issue for the suitep theme.
3. Fixed the code to match our coding standards

## Motivation and Context

My changes ensure that the calendar titles are the users name. Before this change the top user would be mixed up with the administrator, due to the naming convention of each calendars div.

## How To Test This


1. Go to the calendar
2. Got to a shared view
3. Ensure that the calendar settings are set to sepeared by user
4. Ensure that each calendar title correctly identifies the user by hovering over the events in that calendar.

## Types of changes


### Final checklist
